lib/rules/newline-before-return.js

```diff
@@ -75,23 +75,6 @@ module.exports = function(context) {
         }
 
         comments.forEach(function(comment) {
-            // https://github.com/eslint/eslint/pull/5512
```

Comment:
Removed guard that alerted me to this issue in the first place (while keeping the test), as its now unnecessary (yay)